stock_barcodes_gs1/wizard/stock_production_lot.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import models
+
+
+class WizStockBarcodesNewLot(models.TransientModel):
+    _inherit = "wiz.stock.barcodes.new.lot"
+    _description = "Wizard to create new lot from barcode scanner"
+
+    def _decode_barcode(self, barcode):
+        return self.env["gs1_barcode"].decode(barcode)
+
+    def on_barcode_scanned(self, barcode):
+        try:
+            barcode_decoded = self._decode_barcode(barcode)
+        except Exception:
+            return super().on_barcode_scanned(barcode)
+        package_barcode = barcode_decoded.get("01", False)
```

Comment:
Shouldn't the type of barcode depends on nomenclature ?

@sergio-teruel @JasminSForgeFlow

Reply:
Agree, see https://github.com/OCA/stock-logistics-barcode/pull/445